src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditQuestionsUiTest.java

```diff
@@ -10,34 +10,29 @@
 import org.openqa.selenium.WebElement;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import teammates.common.datatransfer.DataBundle;
 import teammates.common.datatransfer.FeedbackParticipantType;
-import teammates.common.datatransfer.FeedbackSessionAttributes;
 import teammates.common.util.Const;
 import teammates.common.util.ThreadHelper;
-import teammates.common.util.TimeHelper;
 import teammates.common.util.Url;
-import teammates.test.driver.AssertHelper;
 import teammates.test.driver.BackDoor;
 import teammates.test.driver.TestProperties;
 import teammates.test.pageobjects.Browser;
 import teammates.test.pageobjects.BrowserPool;
 import teammates.test.pageobjects.FeedbackQuestionSubmitPage;
-import teammates.test.pageobjects.FeedbackSubmitPage;
 import teammates.test.pageobjects.InstructorFeedbackEditPage;
-import teammates.test.pageobjects.InstructorFeedbacksPage;
 import teammates.test.util.Priority;
 
-import com.google.appengine.api.datastore.Text;
-
 /**
- * Covers the 'Edit Feedback Session' page for instructors.
+ * The second test that covers the 'Edit Feedback Session' page for instructors.
+ * This part of the test covers different capabilities of different question types.
  * SUT is {@link InstructorFeedbackEditPage}.
  */
 @Priority(-1)
-public class InstructorFeedbackEditPageUiTest extends BaseUiTestCase {
+public class InstructorFeedbackEditQuestionsUiTest extends BaseUiTestCase {
```

Comment:
Why rename the file? InstructorFeedbackEditPageUiTest follows the naming convention we use for other tests.